templates/layout-slider.php

```diff
@@ -7,27 +7,29 @@
 
 <div class="gallerya gallerya--slider">
   <ul class="js-gallerya-slider js-gallerya-lightbox">
-    <?php foreach ($images as $image):
+    <?php $image_attr = Plugin::lazyLoadIsActive() ? ['data-no-lazy' => '1'] : []; ?>
+    <?php foreach ($images as $index => $image):
       $caption = apply_filters('gallerya/image_caption', $image->post_excerpt, $image->ID);
     ?>
       <li>
         <figure class="gallerya__image">
           <a href="<?= wp_get_attachment_image_src($image->ID, apply_filters('gallerya/image_size_lightbox', 'large'))[0] ?>" <?= !empty($caption) ? 'data-sub-html="' . esc_attr($caption) . '"' : '' ?>>
-            <?= wp_get_attachment_image($image->ID, apply_filters('gallerya/image_size_slider', $slider_image_size)) ?>
+            <?= wp_get_attachment_image($image->ID, apply_filters('gallerya/image_size_slider', $slider_image_size), FALSE, !$index ?: $image_attr) ?>
```

Comment:
```php
!$index ?: $image_attr
```
most likely does not do what you thought it would do.

If you disagree, please confirm with raw debug PHP to prove just that statement on the command line.

Reply:
Checked and you are right. This should be:
```
$index ? [] : $image_attr;
```